cypress/integration/buy-tshirt.spec.ts

```diff
@@ -1,20 +1,29 @@
+import { MenuContentPage,ProductListPage,ProductAddedModal, SummaryPage,SignInPage, AddressPage, ShippingPage, PaymentPage, BankPaymentPage, OrderSummary } from '../page';
```

Comment:
esta linea esta demasiado larga, cambiala asi para que se pueda leer facil:
```suggestion
import { 
  MenuContentPage,
  ProductListPage,
  ProductAddedModal, 
  SummaryPage,
  SignInPage, 
  AddressPage, 
  ShippingPage, 
  PaymentPage, 
  BankPaymentPage, 
  OrderSummary
} from '../page';
```